addons/redis/scripts/redis-sentinel-start-v2.sh

```diff
@@ -42,6 +42,93 @@ build_redis_sentinel_conf() {
   echo "build redis sentinel conf succeeded!"
 }
 
+check_register_sentinel_conf() {
+  if [ -f /data/sentinel/init_done.conf ]; then
+    echo "normal start"
+  else
+    echo "horizontal scaling"
+    if register_sentinel_conf; then
+      touch /data/sentinel/init_done.conf
+    else
+      echo "register_sentinel_conf failed"
+      exit 1
+    fi
+  fi
+}
+
+register_sentinel_conf() {
+    if [[ -n "${SENTINEL_POD_FQDN_LIST}" ]]; then
+        old_ifs="$IFS"
+        IFS=','
+        set -f
+        read -ra sentinel_pod_fqdn_list <<< "${SENTINEL_POD_FQDN_LIST}"
+        set +f
+        IFS="$old_ifs"
+
+        output=""
+        set +e
+        for sentinel_pod_fqdn in "${sentinel_pod_fqdn_list[@]}"; do
+            temp_output=$(redis-cli -h "$sentinel_pod_fqdn" -p 26379 -a "$SENTINEL_PASSWORD" sentinel masters 2>/dev/null)
+            if [[ -n "$temp_output" ]]; then
+                output="$temp_output"
+                break
+            fi
+        done
+        set -e
+    else
+        echo "SENTINEL_POD_NAME_LIST environment variable is not set or empty."
+        return 1
+    fi
+
+    if [[ -n "$output" ]]; then
+        master_name=""
+        master_ip=""
+        master_port=""
+        while read -r line; do
+            case "$line" in
+                name)
+                    read -r master_name
+                    ;;
+                ip)
+                    read -r master_ip
+                    ;;
+                port)
+                    read -r master_port
+                    ;;
+            esac
+
+            if [[ -n "$master_name" && -n "$master_ip" && -n "$master_port" ]]; then
+                echo "Master Name: $master_name, IP: $master_ip, Port: $master_port"
+
+                echo "sentinel monitor $master_name $master_ip $master_port 2" >> /data/sentinel/redis-sentinel.conf
+                echo "sentinel down-after-milliseconds $master_name 5000" >> /data/sentinel/redis-sentinel.conf
```

Comment:
The values of these configurations（like down-after-milliseconds, etc） should also be obtained from the result of the sentinel, rather than resetting them to default values.